Cargo.toml

```diff
@@ -32,6 +31,20 @@ bincode = "1.3.3"
 
 csv = "1.3.0"
 
+dashmap = "5.5.3"
+
+tonic = { version= "0.12.3", features=["gzip"] }
```

Comment:
tonic dependency had some historic version issues. do we need that here?

Reply:
@grooviegermanikus  - I think those dependencies cam from your performancetest branch - I just updated tonic to most recent - we need this to enable gzip in yellowstone_grpc_util

```rust
        GeyserClient::with_interceptor(channel, interceptor)
            .accept_compressed(CompressionEncoding::Gzip)
```

Reply:
lets move forward, i found a workaround for the zeroize version issue.

Reply:
you can drop Gzip support if it helps